landingPage.css

```diff
@@ -0,0 +1,181 @@
+:root {
+    --default-font-color: #fff;
+    --default-font-size: 16px;
+    --description-font-color: #000;
+    --default-background-color:#333;
+    --default-font-family: "Trebuchet MS";
+}
+
+html {
+    color: var(--default-font-color);
+    font-size: var(--default-font-size);
+    font-family: var(--default-font-family);
+}
+
+html, body {
+    height: 100%;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--default-background-color);
+}
+
+body, ul {
+    margin: 0;
+    padding: 0;
+}
+
+/* Navigation Bar */
+
+.nav-top {
+    text-align: center;
+    margin: 0 auto;
+    height: 100px;
+}
+
+.nav-top img {
+    max-width: 96px;
+    height: auto;
+}
+
+.nav-top ul {
+    padding: 50px;
+    margin: 0px;
+    list-style: none;
+    text-align: center;
+    display: inline-block;
+    vertical-align:top;
+}
+
+.nav-top ul li {
+    display: inline;
+    margin: 24px 48px;
+}
+
+.nav-main {
+    text-decoration: none;
+    color: inherit;
+    font-size: var(--default-font-size);
+}
+
+.nav-main:hover {
+    text-decoration: underline;
+}
+
+/* Hero Section */
+.hero {
+    height: 1200px;
+    text-align: center;
+    background-image: url("./Assets/heroHats.jpeg");
+    background-size: cover;
+    background-position: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.hero-text {
+    background-color: rgba(0, 0, 0, 0.4);
+    text-align: center;
+    font-size: calc(var(--base-font)*2);
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.hero-text button {
+    border: none;
+    outline: 0;
+    display: inline-block;
+    padding: 10px 25px;
```

Comment:
Could you keep your paddings consistent (e.g. if you used multiples of 8 maybe not use multiples of 5 here)